### ISSUE TITLE
bo4 just got updated and now the injector no longer works

### ISSUE THREAD

**Issue by CrashedNLanded** [MagicMuffinBot/Magic-Muffin-Bot-Bo4-Unlock-All#3]
bo4 just received a new update that fixed the black market but also the injector no longer is working

**Comment by jambo634** [comment]
yep this hack does not work at all after yesterday bo4 update i am really sad and angry this neeeds to be updated i really wanna use this hack again
this made bo4 so fun

**Comment by jambo634** [comment]
> bo4 just received a new update that fixed the black market but also the injector no longer is working
yeah i am devastated that this no longer works 
this needs update fast

**Comment by tyl0413** [comment]
Broken after latest update.
